recipes/markitdown/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "markitdown" %}
+{% set version = "0.0.1a2" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/markitdown-{{ version }}.tar.gz
+  sha256: b4fb8234e01f743d9a31c5191ae446ccf8220b17ee90598323b185c145faa407
+
+build:
+  entry_points:
+    - markitdown = markitdown.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - hatchling
+    - pip
+  run:
+    - python >={{ python_min }}
+    - beautifulsoup4
+    - requests
+    - mammoth
+    - markdownify
+    - numpy
+    - python-pptx
+    - pandas
+    - openpyxl
+    - pdfminer.six
+    - puremagic
+    - pydub
+    - youtube-transcript-api
+    - speechrecognition
+    - pathvalidate
+    - pdfminer-six
+
+test:
+  imports:
+    - markitdown
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
+    - pip
+
+about:
+  home: https://github.com/microsoft/markitdown
+  summary: Python tool for converting files and office documents to Markdown.
+  dev_url: https://github.com/microsoft/markitdown
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - johnhany97
```

Comment:
@sarthakpati do you also want to be listed as a recipe maintainer for `markitdown`?

Reply:
Sure, thanks!

Reply:
Please do that in the feedstock. Thank you both!